app/src/ui/organism/product-category/ProductCategory.js

```diff
@@ -2,20 +2,13 @@ import Image from "../../atoms/imagine/Image";
 import Category1 from "../../../assets/img/c-img-1.png";
 import CustomLink from "../../atoms/custom-link/CustomLink";
 import CateroryShop1 from "../../../assets/img/c-shop-img-1.png";
+import SectionTitle from "../../molecules/section-title/SectionTitle";
 
 const ProductCategory = () => {
   return (
     <div className="product-category-section">
       <div className="container">
-        <div className="d-flex align-items-center justify-content-between mb-3">
-          <h3 className="fw-bold es-h3-xl">Market Category</h3>
-          <a
-            className="text-decoration-none text-black fw-bold position-relative es-underline-hover"
-            href="#"
-          >
-            View All
-          </a>
-        </div>
+        <SectionTitle title="Market Category" />
         <div className="category-section">
           <div className="d-flex flex-column align-items-center gap-2">
```

Comment:
we can remove this div if you want, just call me :)

Reply:
I am not sure that is good idea